src/seedu/addressbook/commands/Command.java

```diff
@@ -26,6 +26,10 @@ public Command(int targetIndex) {
     protected Command() {
     }
 
+    public boolean isMutating() {
+        return false;
+    }
```

Comment:
Missing header comment. All non trivial classes/methods should have javadoc format header comments. In this case this is a method all sub classes need to implement. Therefore, it should have a good header comment explaining what the method is supposed to do.